CHANGELOG.md

```diff
@@ -5,8 +5,11 @@
         - Plugins supported: `table`
     - API:
       - Support for exits numbers and names. New member `exits` in `RouteStep`, based on `junction:ref` on ways
+      - `RouteStep` now has new parameter `classes` that can be set in the profile on each way.
```

Comment:
If I understand the classes feature correctly the user can set a class per way in the profile (way function) and we will then return it in the response `RouteStep`. Our `RouteStep` does not correspond to ways, so all the classes field tells the user is the road class _for this step_ but not for roads before or after it. Especially with the toll use-case I don't understand how this is useful then. Shouldn't classes be more like an annotation-feature then similar to e.g. osm ids?